exercises/poker/src/main/java/Poker.java

```diff
@@ -0,0 +1,12 @@
+import java.util.List;
```

Comment:
I remove function declarations and just left class declaration as I saw in another exercice that I take for example. Is this okay ?